lib/feed-entrypoints.js

```diff
@@ -7,7 +7,5 @@
  * @returns {object} - First entry in feed with property entry: true
  */
 module.exports = function feedEntrypoint(feed) {
-    for (const item of feed) {
-        if (item.entry) return item;
-    }
+    return feed.filter(item => Boolean(item.entry));
```

Comment:
is `Boolean` needed? it should either be a boolean already or `undefined`

Reply:
yea true. Plowing ahead for now... :plow <-- really no plow emoji?

Reply:
there really should be a plow emoji